### DISCUSSION TITLE
Digital Ocean as MySQL data source provider

### DISCUSSION THREAD

**Discussion by GraceGreen02** [Budibase/budibase#14248] [Help] [answered]
Good day. I am new to Budibase. How can I use a MySQL database on Digital Ocean as a data source in Budibase?

**Comment by ConorWebb96** [comment, marked as answer]
You do this the same way as connecting to any database. Get the IP of the database, password and user + database name and use those to connect to your [database](https://docs.budibase.com/docs/mysql-mariadb).
